qlib/log.py

```diff
@@ -13,12 +13,12 @@
 
 
 class MetaLogger(type):
-    def __new__(cls, name, bases, dict):
+    def __new__(mcs, name, bases, dict_):
```

Comment:
why do you rename `dict` to `dict_` ?
Can you give me any references for this naming convention?

Reply:
To avoid overwriting the python keyword, the suggestion of use _ as postfix comes from the chapter 2-4 in the book Python Tricks.

Reply:
I see now. Thanks to your advice
Will `attrs` be a better name than `dict_`?

Reply:
LGTM for attrs. Will change to it.